core/src/main/java/org/mskcc/oncotree/utils/TumorTypesUtil.java

```diff
@@ -413,6 +412,17 @@ private List<TumorType> findTumorType(TumorType allTumorTypes, TumorType current
                                 StringUtils.containsIgnoreCase(currentTumorType.getMainType(), keyword));
                 }
                 break;
+            case "clinicalcasessubset":
```

Comment:
This actually is not needed.
I believe that we should not expose the clinicalCasesSubset identifiers to the users. These are internal identifiers which we do not want users to use or become dependent on, so I think they should not appear in the oncotree website, or in any response from the oncotree webserver api.

So we should not support searching the oncotree based on these codes either.

Reply:
removed